server/src/Controllers/GameController.ts

```diff
@@ -46,15 +46,28 @@ export class GameController {
   };
 
   createGame = async (req: Request, res: Response) => {
-    const { userId, gameName, primary_color, secondary_color, tertiary_color } =
-      req.body;
-      const csvFilePath = req.file?.path!;
+    const { name, primaryColor, secondaryColor, tertiaryColor } = req.body;
+    const userId = req.user?.id!;
```

Comment:
this endpoint goes through auth middleware now so we can grab the `id` of the user as opposed to having to pass it in